src/subcommand.rs

```diff
@@ -85,9 +85,12 @@ impl Subcommand {
 
     /// Reads gz mrt data from urls defined by range, decompresses them, parses mrt output, finds bottleneck.
     fn find_bottleneck(dump: &PathBuf, out: Option<&Path>) -> Result<()> {
-        let bottleneck = FindBottleneck::locate(dump)?;
-        bottleneck.write(out)?;
-
+        // TODO handle errors
+        fs::remove_dir("tmp");
```

Comment:
For handling of all temporary files and directories, lets use the [tempfile crate](https://docs.rs/tempfile/3.1.0/tempfile/).